### DISCUSSION TITLE
How do you make these awesome release notes?

### DISCUSSION THREAD

**Discussion by jbusecke** [intake/intake-esm#319] [Q&A] [answered]
I love the [intake-esm release notes](https://github.com/intake/intake-esm/releases). Are these generated automatically somehow? Or is that all manual?

**Comment by andersy005** [comment, marked as answer]
I use this package: https://github-activity.readthedocs.io/en/latest/. If you have a compatible [PR labeling system](https://github-activity.readthedocs.io/en/latest/#splitting-prs-by-tags-and-prefixes), github-activity may be able to do 95% of the work. The remaining 5% (adjusting titles or adding emojis) can be done manually.

**Comment by jbusecke** [comment]
Ohhh awesome. Ill def take a look!
